### PR TITLE
Skip automatic API review for management packages

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Skip automatic review creation for management packages since these are not currently reviewed by architects

### CODE REVIEW

**Review by scbedd — APPROVED** [review]

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -185,6 +185,13 @@ function Find-python-Artifacts-For-Apireview($artifactDir, $artifactName)
 {
   # Find wheel file in given artifact directory
   # Make sure to pick only package with given artifact name
+  # Skip auto API review creation for management packages
+  if ($artifactName -match "mgmt")
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:189]: Can we please get confirmation from some one on the mgmt team so they are aware this is happening and can confirm this is what they want?

Comment (praveenkuttappan) [reply eng/scripts/Language-Settings.ps1:189]: I have discussed this with Anna and Johan about this and making this change as per their request. Will definitely send a note to mgmt team also. I have a call with management team tomorrow. But I would like to disable auto reviews for the time being for management plane until they agree to approve reviews.

Comment (weshaggard) [reply eng/scripts/Language-Settings.ps1:189]: I just want to be sure we are including them in the conversation and not just making decisions on their behalf. If it is easier to disable them for now that is fine just be sure to start the conversation with them.

**Pull request merged**